### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by donice**
In this PR, I:
- I implented the About and Index HTML files
- Styled them in their respective css files
- Added the features dynamic display functionalities in the Js file
- Implemented the dynmic nav in the Js file
- Repeatedly cleared linter erros

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,20 +2,210 @@
...
+      <ul class="nav-desktop">
+        <li><a href="./about.html" >About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#join">Join</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+        <li><a href="#news">News</a></li>
+        <li><a href="#campaign" id="campaign">CC Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
```
Comment (andersonlebon) [inline index.html]: - [ ]  As the design requires we should keep spaces between elements, kindly make sure the you use the same thing for your desketop menu items. Check this

Your design | the template
---|---
image | image


- [ ] I think there is a missing tob navigation bar, kindly add it as required in the design guidlines. 

image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        </a>
+
+        <ul class="nav-desktop">
+          <li><a href="#about">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#join">Join</a></li>
+          <li><a href="#sponsor">Sponsor</a></li>
+          <li><a href="#news">News</a></li>
+          <li><a href="#campaign" id="campaign">CC Campaign</a></li>
+        </ul>
```
Comment (andersonlebon) [inline about.html]: - [ ] KIndly provide the link to the home page in the about page navigation bar as required

File: about.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        <div class="program-header">
+          <h2>
+            Software Developers <br />
+            Global Summit 2023 Logo
+          </h2>
+          <div class="program-underline"></div>
+          <p>
+            The logo of CC Global Summit 2015 was decided through the logo
+            competition from 8, June to 7, July.
+          </p>
```
Comment (andersonlebon) [inline about.html:129]: - [ ] KIndly make sure your design match with the template, this fist section should have a backgound image abd a transparent color.

- [ ] Remember that your project should have the same design as the template but you should presonalize you content with a good logic by choosing a topic different to the template one.

Your design | the template
---|---
image | image



### Apply the same review to the About page

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @donice 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Highlights 🍾 

- Nice design 👏  ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @donice 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organization ✔️
- [x] Nice use of CSS features ✔️ 
- [x] Kudus on meeting the requirements suggested  ✔️


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**